### DISCUSSION TITLE
Suggestion for "Shape Grid Transition"

### DISCUSSION THREAD

**Discussion by PatLabGit** [lbalazscs/Pixelitor#362] [Ideas]
The filter is at `Filter > Transitions > Shape Grid Transition...`

I suggest a check box to "Invert" the opaque and the alpha
For example actually it make transparent circle like this

![screenshot_20240408-171741](https://github.com/lbalazscs/Pixelitor/assets/47497693/f6b6729e-e9d6-49f4-84dc-dad421640d9c)

If we can invert between the opaque and the transparency, we could make dotted map or dotted people, dotted whatnot, something like this

![screenshot_20240408-171520](https://github.com/lbalazscs/Pixelitor/assets/47497693/64acbfef-255c-4e63-89a6-569e3fa1b6bf)

**Comment by lbalazscs** [comment]
Good idea! In the newest commit "Shapes Grid Transition", "Checkerboard Transition", and "Goo Transition" have a new "Invert Transparency" checkbox. (there were some technical difficulties with "2D Transition" and "Blinds Transition", so I left them out)

**Reply by PatLabGit** [reply]
Thank you!

**Comment by LinuxBeaver** [comment]
This is already possible using the erase blend mode with shapes grid transition.

**Reply by lbalazscs** [reply]
There are many ways to achieve such an effect. You can use the erase blend mode, as you said, or you can create a layer mask from transparency, invert it, and then use it on the original image. But none of these tricks is as simple as clicking an "Invert Transparency" checkbox...

**Reply by PatLabGit** [reply]
@LinuxBeaver  the mask or the Erase blend mode are 2 nice tips to know about, but a beginner will struggle by lack of knowledge.

Yesterday (I did play a lot with Pixelitor), while using this filter I just told myself "_it would be nice to have an Invert option_".
A simple click from the user can remove a lot of frustration for a beginner > https://www.gimp-forum.net/Thread-currently-planning-to-create-a-dotted-map-of-the-world-any-ideas-here 
It's an exceptionally long thread where we can see the OP struggling to make a dotted map, now with Pixelitor, one click... ;)
